tests/test_interpImageTask.py

```diff
@@ -122,6 +122,7 @@ def validateInterp(miInterp, useFallbackValueAtEdge, fallbackValue):
                         # Should raise if negative not allowed, but USER supplied negative value
                         if not negativeFallbackAllowed and fallbackValueType == "USER":
                             self.assertRaises(ValueError, config.validate)
+                            continue
```

Comment:
How did this work before?

Reply:
Since the validation wasn't being run before, it just continued on into the rest of that iteration with a non-valid config. I don't know what, if any, failure modes it might have had, but clearly it didn't fail.